doc/concepts/configuration/configuration_connections.rst

```diff
@@ -0,0 +1,292 @@
+These settings might include the credentials required to connect to this instance, a URI used to listen for incoming requests, and SSL settings.
+
+If an advertise URI is not specified explicitly, a :ref:`listen URI <configuration_connections_listen_uri>` of this instance is used.
+In this case, you need at least to specify credentials for connecting to this instance.
+
+..  NOTE::
+
+    If several listen URIs are specified, the first one is used as an advertise URI.
+
+.. _configuration_connections_advertise_credentials:
+
+Connection credentials
+~~~~~~~~~~~~~~~~~~~~~~
+
+In the example below, the ``iproto.advertise.peer`` option is used to inform other replica set members that the ``replicator`` user should be used to connect to the current instance:
+
+..  literalinclude:: /code_snippets/snippets/replication/instances.enabled/advertise_peer/config.yaml
+    :language: yaml
+    :start-after: roles: [replication]
+    :end-at: login: replicator
+    :dedent:
+
+In a sharded cluster, ``iproto.advertise.sharding`` specifies that a router and rebalancer should use the ``storage`` user to connect to storages:
+
+..  literalinclude:: /code_snippets/snippets/sharding/instances.enabled/sharded_cluster/config.yaml
+    :language: yaml
+    :start-at: iproto
+    :end-at: login: storage
+    :dedent:
+
+
+.. _configuration_connections_advertise_explicitly:
+
+Advertise URI
+~~~~~~~~~~~~~
+
+If required, you can specify an advertise URI explicitly by setting up the :ref:`iproto_advertise.\<peer_or_sharding\>.uri <configuration_reference_iproto_advertise.peer_sharding.uri>` option.
+In the example below, ``iproto.listen`` includes two URIs that can be used to connect to ``instance001`` but only the second one is used to advertise this instance to other replica set peers:
+
+..  literalinclude:: /code_snippets/snippets/replication/instances.enabled/advertise_peer/config.yaml
+    :language: yaml
+    :start-at: instance001:
+    :end-before: instance002:
+    :dedent:
+
+The ``iproto_advertise.<peer_or_sharding>.uri`` option can also accept an FQDN instead of an IP address:
+
+..  code-block:: yaml
+
+    instance001:
+      iproto:
+        listen:
+        - uri: '192.168.0.101:3301'
+        advertise:
+          peer:
+            uri: 'server001.example.com:3301'
+
+
+.. _configuration_connections_advertise_uri_ssl:
+
+SSL parameters
+~~~~~~~~~~~~~~
+
+:ref:`SSL parameters <configuration_connections_ssl>` for an advertise URI should be set only if this :ref:`advertise URI is specified explicitly <configuration_connections_advertise_explicitly>`.
+Otherwise, SSL parameters of a listen URI are used and no additional configuration is required.
+
+Configuring an advertise URI's SSL options depends on whether a trusted certificate authorities (CA) file is set or not.
+Without the CA file, you only need to set ``iproto_advertise.<peer_or_sharding>.params.transport`` to ``ssl`` as shown below:
```

Comment:
I'm a bit in doubts about 'with/without the CA file' wording. Can we say something like the following.

> Verification of a client certificate on the server is enabled by setting the `ssl_ca_file` parameter in an `iproto.listen` URI. The verification checks that the client certificate issued by the given certificate authority.
>
> Verification of server's certificate is enabled <...>

After this we can just say, whether server/client certificate verification is enabled instead of technical 'with/without the CA file'.